common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/resetpassword/ResetPasswordSubmitApiResponse.kt

```diff
@@ -36,13 +36,12 @@ import java.net.HttpURLConnection
  */
 class ResetPasswordSubmitApiResponse(
     @Expose override var statusCode: Int,
-    @SerializedName("password_reset_token") val passwordResetToken: String?,
+    @SerializedName("continuation_token") val continuationToken: String?,
     @Expose @SerializedName("poll_interval") val pollInterval: Int?,
-    @SerializedName("error") val error: String?,
-    @SerializedName("error_description") val errorDescription: String?,
-    @SerializedName("error_uri") val errorUri: String?,
-    @SerializedName("details") val details: List<Map<String, String>>?,
-    @SerializedName("inner_errors") val innerErrors: List<InnerError>?
+    @Expose @SerializedName("error") val error: String?,
```

Comment:
`error` and other fields should not be exposed as they might contain PII. 
Can you go through the API response classes and make sure you're not introducing `@Expose` where it shouldn't be introduced?

Reply:
The error field contains just the error value from the API I don't think it could ever contain PII. I will have another look over the rest of the response classes